examples/cinder-csi-plugin/clone/source-pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: source-pvc
+spec:
+  storageClassName: csi-cinderplugin-sc
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
maybe other than those examples, you can add some info in the doc as well?

Reply:
sure, Ill add in separate PR ?

Reply:
works for me ,thanks